.gitignore

```diff
@@ -91,3 +91,4 @@ FMODAssets/.user/
 # If the source bank files are kept outside of the StreamingAssets folder then these can be ignored.
 # Log files can be ignored.
 fmod_editor.log
+Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset
```

Comment:
https://qa.fmod.com/t/fmodstudiosettings-asset-keeps-appearing-in-git-without-changing-anything/17318